js/script.js

```diff
@@ -1112,11 +1113,29 @@ var validCombos = {
         "Download .tgz file from&nbsp;<a href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a>",
 
     "mac,Java,ARM64,CoreML":
-      "Download .tgz file from&nbsp;<a href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a>",
+        "Add a dependency on <a href='https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime' target='_blank'>com.microsoft.onnxruntime:onnxruntime</a> using Maven/Gradle",
     
     "mac,C-API,ARM64,CoreML":
       "Download .tgz file from&nbsp;<a href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a>",
 
+    "mac,Python,ARM64,DefaultCPU":
+        "pip install onnxruntime",
+    
+    "mac,Python,ARM64,DefaultCPU":
+        "pip install onnxruntime",
+
+    "mac,Java,ARM64,DefaultCPU":
+        "Add a dependency on <a href='https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime' target='_blank'>com.microsoft.onnxruntime:onnxruntime</a> using Maven/Gradle",
+
+    "mac,C#,ARM64,DefaultCPU":
+        "Install Nuget package&nbsp;<a href='https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime' target='_blank'>Microsoft.ML.OnnxRuntime</a>",
+
+    "mac,C-API,ARM64,DefaultCPU":
```

Comment:
Would be better to make it as the same "mac,C-API,ARM64,CoreML" because we only have one package for "mac,C-API,ARM64", which includes both the default CPU EP and the CoreML EP.

Reply:
Should both point to the Nuget package then?

Reply:
I think either is ok. I don't have a strong opinion on this. You may consult the other devs.